stack.py

```diff
@@ -0,0 +1,59 @@
+from static_array import StaticArray
+
+
+class Stack:
+    def __init__(self, size):
+        self.size = size
+        self.array = StaticArray(size)
+        self.occupied_size = 0
+
+    def push(self, element):
+        if self.occupied_size == self.size:
+            self.array.insert(element, self.occupied_size)
+            self.occupied_size += 1
+            return True
+        return False
+
+    def pop(self):
+        if self.occupied_size == 0:
+            return False
+        self.array.delete(self.occupied_size)
+        self.occupied_size -= 1
+
+    def peak(self):
+        if self.occupied_size == 0:
+            return None
+        return self.array.get(self.occupied_size)
```

Comment:
IF im trying to `peek` value when stack is empty, should it return None or throw error ? What makes more sense ?

Reply:
I feel None should be fine because it's one of the possible scenario. So instead of error, we can gracefully return None

Reply:
Makes sense